Networking/Networking/Remote/SiteRemote.swift

```diff
@@ -53,6 +57,12 @@ public class SiteRemote: Remote, SiteRemoteProtocol {
 
         return try await enqueue(request)
     }
+
+    public func launchSite(siteID: Int64) async throws {
+        let path = "sites/\(siteID)/launch"
```

Comment:
Nit: Consider moving this to the `Path` enum with a static function that takes siteID as a parameter.